### PR TITLE
All Digests Page

### PR DESCRIPTION
**Pull request by ShevtsovAnton**
[Trello task - User story: All digests page](https://trello.com/c/YnOuCwSd/54-user-story-all-digest-page)

### CODE REVIEW

File: components/Digests/DigestItem.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        <StyledHeader>
+          <Icon iconType={IconTypes.Date} size={IconSizes.Medium} isActive={true} />
+          <Text className="digest-date">
+            {item?.digest_date && format(new Date(item.digest_date), 'dd.MM.yy')}
+          </Text>
+          <TextH3 className="digest-number">Digest 11</TextH3>
+        </StyledHeader>
+        <StyledContent>
+          <div className="item-image">
+            <Image src=""/>
```
Comment (OlegSymonchuk) [inline components/Digests/DigestItem.tsx]: Why not used a real src value from DB?

File: components/Digests/DigestItem.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+const DigestItem = ({ item }: DigestItemProps) => {
+  return (
+    <Link href={`/digests/${item?.id}`}>
+      <Wrapper>
+        <StyledHeader>
+          <Icon iconType={IconTypes.Date} size={IconSizes.Medium} isActive={true} />
+          <Text className="digest-date">
+            {item?.digest_date && format(new Date(item.digest_date), 'dd.MM.yy')}
+          </Text>
+          <TextH3 className="digest-number">Digest 11</TextH3>
```
Comment (OlegSymonchuk) [inline components/Digests/DigestItem.tsx]: Is it correct to use a static text instead of a value?

File: components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    setIsTabletMenuDisplayed(!isTabletMenuDisplayed);
+  };
+
+  return (
+    <header>
+      <Wrapper>
+        <div className="top-row">
+          <Link href={'/'}>
+            <a>
+              <Logo>
```
Comment (OlegSymonchuk) [inline components/Header/Header.tsx]: There is the Logo component.

Comment (ShevtsovAnton) [reply components/Header/Header.tsx]: added <Logo/> component
We need to investigate a warning in the console. 
Warning: Prop 'fill' did not match. Server: '...' Client: '....'
Seems to be related to changing Logo colors.

File: components/Digests/DigestItem.tsx (hunk 2 above)
Comment (ShevtsovAnton) [reply components/Digests/DigestItem.tsx]: Left as is for now, added @todo, we need to determine how we assign numbers to digests.

User story: 
 Digest card displays all the information about it (Image/Title (in a format: 'Digest №__')/Date (of communication to users)/Description)

File: components/Digests/DigestItem.tsx (hunk 1 above)
Comment (ShevtsovAnton) [reply components/Digests/DigestItem.tsx]: added url of the first image in posts array. Added @todo, need to handle the case, when the url returns undefined value.

**Review by OlegSymonchuk — APPROVED** [review]

**Pull request merged**